src/examples/java/io/nats/examples/ExampleArgs.java

```diff
@@ -162,77 +166,83 @@ public static Builder builder() {
     }
 
     public static class Builder {
-        private String subject;
-        private String queue;
-        private String message;
-        private int msgCount = -1;
-        public int subCount = -1;
-        private String stream = null;
-        private String consumer = null;
-        private String durable = null;
-        private String deliver = null;
-        private int pullSize = 0;
-        private String user = null;
-        private String pass = null;
+        private String subjectDefault;
+        private String queueDefault;
+        private String messageDefault;
+        private int msgCountDefault = -1;
+        public int subCountDefault = -1;
+        private String streamDefault = null;
+        private String mirrorDefault = null;
+        private String consumerDefault = null;
+        private String durableDefault = null;
+        private String deliverDefault = null;
+        private int pullSizeDefault = 0;
+        private String userDefault = null;
+        private String passDefault = null;
```

Comment:
made these variable names less ambiguous. when my own code confuses me, I had to change it